cmd/soroban-cli/src/commands/network/mod.rs

```diff
@@ -6,17 +6,21 @@ use super::{config::locator, global};
 
 pub mod add;
 pub mod container;
+pub mod default;
 pub mod ls;
 pub mod rm;
 
 #[derive(Debug, Parser)]
 pub enum Cmd {
     /// Add a new network
     Add(add::Cmd),
+
```

Comment:
Why so many new empty lines? Is this personal preference?

Reply:
Because it's very hard to quickly read the file when you have one giant blob without breaklines around specific concerns (the effort is too big).

But yeah, def a personal preference.